src/terminal/parser/stateMachine.cpp

```diff
@@ -254,7 +254,7 @@ static constexpr bool _isOscTerminationInitiator(const wchar_t wch) noexcept
 // - True if it is. False if it isn't.
 static constexpr bool _isOscInvalid(const wchar_t wch) noexcept
 {
-    return wch <= L'\x17' ||
+    return (wch <= L'\x17' && wch != L'\x0a' && wch != L'\x0d') ||

```

Comment:
Isn't this redundant?

Reply:
Actually, this wasn't. Before I added base64 support, to support passing "\r\n" into the clipboard, the state machine should view `\r 0x0d` and `\n 0x0a` as valid OSC characters. But `wch <= L'\x17'` will make them invalid, so I changed it to `(wch <= L'\x17' && wch != L'\x0a' && wch != L'\x0d')`.

Anyway, that change is no longer needed, because base64-encoded string won't include `\r` and `\n`.